src/components/ImageView/index.js

```diff
@@ -3,11 +3,16 @@ import PropTypes from 'prop-types';
 import {
     View, Image, Pressable,
 } from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import FastImage from '../FastImage';
```

Comment:
@thomas-coldwell Any specific reason we didn't import `FastImage` to `ImageView/index.native.js`? It imports [Image](https://github.com/Expensify/App/blob/0294e1c130f1bae588abcb0516cd3d221c300d1a/src/components/ImageView/index.native.js#L5-L7) from `react-native-fast-image` itself, but it would be better to have consistent imports?

Reply:
I've fixed this now - accidentally missed the native file and didn't notice during my own testing - thanks for catching this! I've set it up to also use `FastImage` from the custom component wrapper I've created rather than the direct library import in the native implementation file